File: swing/src/main/java/wannabe/swing/renderer/FilledCircle.java
package wannabe.swing.renderer;

import java.awt.Graphics;
import wannabe.swing.SwingProjected;

public class FilledCircle extends SwingRenderer {
  @Override public void draw(Graphics g, SwingProjected p) {
    g.fillOval(p.left, p.top, p.size, p.size);
  }
}

File: swing/src/main/java/wannabe/swing/renderer/FilledCircle.kt
package wannabe.swing.renderer

import wannabe.swing.SwingProjected
import java.awt.Graphics

class FilledCircle : SwingRenderer() {
  override fun draw(
    g: Graphics,
    p: SwingProjected
  ) {
    g.fillOval(p.left, p.top, p.size, p.size)
  }
}
